bower.json

```diff
@@ -34,7 +34,7 @@
     "web-component-tester": "^4.0.0",
     "webcomponentsjs": "webcomponents/webcomponentsjs#0.7.0 - 1",
     "test-fixture": "PolymerElements/test-fixture#1.0.0 - 3",
-    "imagecapture-polyfill": "https://unpkg.com/image-capture@0.3.2/lib/imagecapture.js",
+    "imagecapture-polyfill": "https://unpkg.com/image-capture@^0.3.2/lib/imagecapture.js",
```

Comment:
Since this polyfill does not yet have a major release, let's use `~0.3.2` instead.